Pinta.Gui.Widgets/Dialogs/SimpleEffectDialog.cs

```diff
@@ -67,14 +67,15 @@ public class SimpleEffectDialog : Gtk.Dialog
 
 		public object EffectData { get; private set; }
 
-		public event PropertyChangedEventHandler EffectDataChanged;
+		public event PropertyChangedEventHandler? EffectDataChanged;
 
 		protected override void Dispose (bool disposing)
 		{
 			// If there is a timeout that hasn't been invoked yet, run it before closing the dialog.
-			if (disposing && event_delay_timeout_id != 0) {
+			if (event_delay_timeout_id != 0)
```

Comment:
I think this might have been merged incorrectly? I think either the `disposing` check should still be there, or this code should reset `event_delay_timeout_id`

Reply:
Doh! Pushed a fix.